src/widgets/index.tsx

```diff
@@ -73,6 +76,12 @@ export const WidgetRoot = memo(({lastUpdate}: {lastUpdate: number}) => {
         screen: 'home',
         uid,
         chainId: app.provider.cosmosChainId,
+        platform: Platform.select({
```

Comment:
https://reactnative.dev/docs/platform#os
```suggestion
        platform: Platform.OS
```

macos doesn't supported by default